docs/Config.md

```diff
@@ -0,0 +1,277 @@
+# **Environment variable configuration**
+
+Some environment variables commonly used for pipelines under osde2e are indicated below. These flags or variables are shared by both osde2e and osde2ectl.
+
+
+## Common Environment Variables
+
+```
+Cluster related:-
```

Comment:
Could each of these subheadings be broken out rather than one large code block? Kind of like 

`### Cluster related:` and then start the new block of config options